### PR TITLE
Clarify the use of OTP GPIO boot modes

### PR DESCRIPTION
**Pull request by pelwell**
Clarify the use of OTP GPIO boot modes, with warnings about the irreversibility of the settings and the bugs in older firmware.

### CODE REVIEW

**Comment by pelwell** [comment]
@JamesH65 Please return to copy edit state until comments are addressed.

**Comment by JamesH65** [comment]
@JaninaPi Hi Janina, couple of mistakes crept in to the CE, please see Phil's comments.

**Comment by JaninaPi** [comment]
I'm sorry about that. Fixed the typo and reformatted one instance of `config.txt`, should all be OK now.

File: hardware/raspberrypi/bootmodes/bootflow.md [hunk 1]
```diff
@@ -35,9 +39,9 @@ Next the boot ROM checks each of the boot sources for a file called bootcode.bin
 
 NOTES: 
 
-* If there is no SD card inserted, the SD boot mode takes five seconds to fail. To reduce this and fall back to USB more quickly, you can either insert an SD card with nothing on it or use the `program_gpio_bootmode` OTP to only enable USB.
+* If there is no SD card inserted, the SD boot mode takes five seconds to fail. To reduce this and fall back to USB more quickly, you can either insert an SD card with nothing on it or use the GPIO bootmode OTP setting descibed above to only enable USB.
```
Comment (pelwell) [inline hardware/raspberrypi/bootmodes/bootflow.md]: Typo: descibed

**Comment by JamesH65** [comment]
Reads OK to me, @pelwell ?

File: hardware/raspberrypi/bootmodes/bootflow.md (hunk 1 above)
Comment (ghollingworth) [reply hardware/raspberrypi/bootmodes/bootflow.md]: Good by me

**Pull request merged**